src/main/java/dev/jbang/cli/BaseBuildCommand.java

```diff
@@ -83,8 +83,17 @@ void setJavaVersion(String javaVersion) {
 	@CommandLine.Option(names = { "--cp", "--class-path" }, description = "Add class path entries.")
 	List<String> classpaths;
 
+	@CommandLine.Option(names = {
+			"-f", "--fresh" }, description = "Force script upate. A new jar is created.", defaultValue = "false")
+	boolean forceUpdate;
```

Comment:
```suggestion
	boolean fresh;
```

lets not mix terms if we can avoid it.